packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -69,7 +73,17 @@ export class WaferMap extends FoundationElement {
     /**
      * @internal
      */
-    @observable public canvasSideLength?: number;
+    public readonly rect!: HTMLElement;
+
+    // Last selectedDie accessor
+    public get lastSelectedDie(): WaferMapDie | undefined {
```

Comment:
It's worth pausing to discuss this new public element API. Some questions/thoughts:
1. naming: what does the "last" part of the name mean? I'd expect to be able to access the current selection (or undefined if nothing is selected) but "last" to me implies something happened in the past. How about just `selectedDie`?
2. API type: is a property getter of type `WaferMapDie` appropriate? This can't be accessed as an HTML attribute and doesn't provide any notification when selection changes so frameworks like Angular can't data bind to it. If we wanted to make it an attribute with a changed event we could change its type to be a number that indexes into the `dies` array.
3. it'd be good to update these places to document this API:
   - API section of the spec
   - storybook for the wafer map
   - if documenting it inline, use a JSDoc style coment `/**` and provide more detail (or just delete the comment above)

Reply:
We could also remove this for now and have this discussion on a future PR, since it looks like this is just staging work for the hover feature.

Reply:
I removed it from the index.ts file. However I left it inside the `EventHandler` and renamed to `selectedDie` - I do not see any reason why it couldn't be named this way. The code will be finalized in the following PR.

Reply:
Maybe I'm not following or maybe you haven't pushed some local changes, but I still see the old API here.

Reply:
It's definitely not there anymore :)